bigquery/deploy/bq-indexer.yaml.templ

```diff
@@ -1,31 +1,36 @@
-apiVersion: batch/v1
-kind: Job
+apiVersion: batch/v1beta1
```

Comment:
Have you confirmed this works? Maybe temporarily set to run every 5 mins.

Let me know what project you tested on, I want to see what this looks like in GCP console.